### ISSUE TITLE
Unhelpful error when missing CNB_USER_ID

### ISSUE THREAD

**Issue by squee1945** [buildpack/pack#149]
When missing `CNB_USER_ID` you get the following when running `pack build`:

```
ERROR: parsing pack uid: : strconv.Atoi: parsing "": invalid syntax
```

**Comment by mgibson1121** [comment]
@ekcasey @djoyahoy Which file does the field `CNB_USER_ID` live in?

**Comment by ekcasey** [comment]
@mgibson1121 It is an environment variable that is set in builder and stack images. To accept this you would need to create an invalid builder (probably using a Dockerfile). For example, you could start with the vanilla `ubuntu:bionic` image and add a stack label but not add the `CNB_USER_ID` env var.

**Comment by ekcasey** [comment]
fixed on `master`, will ship in `v0.2.0`
